src/main/scala/nl.knaw.dans.easy.properties/app/graphql/types/MutationType.scala

```diff
@@ -66,6 +67,14 @@ trait MutationType {
     astDirectives = Vector.empty,
     astNodes = Vector.empty,
   )
+  private val originOptionInputField: InputField[Option[Origin.Value]] = InputField(
```

Comment:
Not sure if these types line up. This has `Option[Origin]`, while below you do `.asInstanceOf[Origin]`.